src/coreclr/src/jit/gentree.cpp

```diff
@@ -10206,10 +10210,12 @@ void Compiler::gtDispRegVal(GenTree* tree)
     }
 
 #if FEATURE_MULTIREG_RET
-    if (tree->IsMultiRegCall())
+    if (tree->OperIs(GT_CALL))
```

Comment:
the similar check under `tree->IsCopyOrReload()` `if (op1->IsMultiRegCall())`  could fail if `ReturnTypeDesc ` not initialized.

Reply:
As it happens, we never have a `COPY` or `RELOAD` until after register allocation, at which time it will always be initialized, but I've changed it, as it is just as effective without the check for `IsMultiRegCall`